include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -53,7 +53,8 @@ static const char * action_color(libdnf::transaction::TransactionItemAction acti
         case libdnf::transaction::TransactionItemAction::REMOVE:
             return "red";
         case libdnf::transaction::TransactionItemAction::REPLACED:
-            return "gray";
+            // TODO(dmach): replace with libdnf::cli::tty::dim?
+            return "\033[2m";
```

Comment:
It can be replaces by "halfbright".

Reply:
@dmach 
So when I read the commit message "Dim removed packages in the transaction table" and "The removed packages are less significant than the others", I said no. This is important information.
But there is something else in the code. The color of `REPLACED` was changed. `REMOVED` stays red. So OK.
Please change the commit message. It is misleading.

Reply:
"\033[2m" == "halfbright"